godtools/App/Share/Data/MobileContentAuthTokenRepository/MobileContentAuthTokenRepository.swift

```diff
@@ -35,7 +37,13 @@ class MobileContentAuthTokenRepository {
         }
     }
     
-    func fetchRemoteAuthToken(oktaAccessToken: String) -> AnyPublisher<String?, URLResponseError> {
+    func fetchRemoteAuthToken(oktaAccessToken: String? = nil) -> AnyPublisher<String?, URLResponseError> {
+        
+        // TODO: -  `getAccessTokenFromPersistentStore` should be updated to `withFreshTokenPerformAction` based on https://github.com/okta/okta-oidc-ios/blob/e5450a3aab5c194a7470addeef176e769a374650/Sources/AppAuth/OKTAuthState.m#L90-L93
```

Comment:
Hey @rachaelblue for this we do have a method available from the ```OktaAuthentication``` class which ```CruOktaAuthentication``` class derives from.  There is a publisher ```cruOktaAuthentication.renewAccessTokenPublisher()```.
I'm fine if we create another task for this.